LLHInterface.h

```diff
@@ -60,8 +80,30 @@ class ReceiveLLHOverBifrost : public TemplateLLHGetter
     fPars = oscpars;
   }
 
+  //Setting Asimov parameters
+  void SetAsimovSystParameters(CovTypes iCov, std::vector<double> vals)
+  {
+    if(fAsimovSysts.size() <= iCov) fAsimovSysts.resize(iCov+1);
+    fAsimovSysts[iCov] = vals;
+  }
+
+  void SetAsimovOscParameters(OscPars oscpars)
+  {
+    fAsimovOscPars = oscpars;
+    fSendAsimovOsc = true;
+  }
+
   double GetLikelihood()
   {
+
+    fBF << int(fSendAsimovOsc);
+
+    if(fSendAsimovOsc) fBF << fAsimovOscPars;
+    int syst_size = fAsimovSysts.size();
+
+    fBF << syst_size;
```

Comment:
I would combine these lines, just so we don't have this extra variable floating around.
```suggestion
    fBF << int(fAsimovSysts.size());
```